src/client/pages/ConnectPayment/components/TrustlyModal.tsx

```diff
@@ -77,53 +80,34 @@ export const TrustlyModal: React.FC<Props> = ({
   }, [])
 
   return (
-    <Container<State, Actions>
```

Comment:
Whoop whoop, nice refactoring! 🤩  Would be nice to get rid of constate overall. 
Just me being picky, but I think it's handy to keep the refactoring in a commit by it's own 🙃

Reply:
I'd rather argue it's better to do it in small chunks in case something breaks. Thoughts on that? :)